p4_symbolic/ir/ir.proto

```diff
@@ -31,18 +31,13 @@ syntax = "proto3";
 
 package p4_symbolic.ir;
 
+import "p4/v1/p4runtime.proto";
 import "p4_pdpi/ir.proto";
 import "p4_symbolic/bmv2/bmv2.proto";
 
-// Table entry parsed from P4RT/simple_switch_CLI.
-message TableEntry {
-  // Name of the target action, matches the format of the key
-  // in P4Program#actions.
-  string action = 1;
-  // Values to match on.
-  repeated int32 match_values = 2;  // TODO(babman): turn to hex for bitstrings.
-  // Parameters passed to action if the match succeeds.
-  repeated int32 action_parameters = 3;  // TODO(babman): turn to hex.
+// Table entry file consisting of many entries in P4RT format.
+message TableEntriesFile {
+  repeated p4.v1.TableEntry table_entries = 1;
```

Comment:
This is pretty much useless from a logical perspective. I added it so that it is easy (a one liner) to parse a file consisting of many consecutive table entries. See  p4-samples/port-table/entries.pb.txt  above for an example.

Something like this may be already built somewhere, but I could not find it in p4-pdpi or p4runtime protos.